x/ecocredit/client/testsuite/grpc.go

```diff
@@ -159,51 +131,29 @@ func (s *IntegrationTestSuite) TestGetBatches() {
 			resp, err := rest.GetRequest(tc.url)
 			require.NoError(err)
 
-			var batches ecocredit.QueryBatchesResponse
-			err = val.ClientCtx.Codec.UnmarshalJSON(resp, &batches)
-
-			if tc.expErr {
-				require.Error(err)
-				require.Contains(string(resp), tc.errMsg)
-			} else {
-				require.NoError(err)
-				require.NotNil(batches.Batches)
-				require.Len(batches.Batches, tc.numBatches)
+			var res core.QueryBatchesResponse
+			err = val.ClientCtx.Codec.UnmarshalJSON(resp, &res)
+			require.NoError(err)
+			require.NotNil(res.Batches)
+			require.Greater(len(res.Batches), 0)
```

Comment:
I think we should include `expItems` rather than `> 0`.

Reply:
see above